daemon-shared/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "daemon-shared"
```

Comment:
naming suggestions: "bin-shared", "app-shared"?  It would be more self-descriptive, as we're sharing the code among binaries, as you described below

Reply:
Yeah good call!